components/centraldashboard/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "kubeflow-centraldashboard",
+  "version": "0.4.0",
```

Comment:
Let's use a seperate versioning for this component, doesn't make sense to link it to Kubeflow-Core's version.

Let's go with `v0.0.1`?

Reply:
SGTM.